README.md

```diff
@@ -70,6 +70,8 @@ You can access these pages on your computer using one of the following clients:
 
 Let us know if you are building one and we can add it to this list!
 
+There is also a comprehensive [list of clients in Wiki](https://github.com/tldr-pages/tldr/wiki/TLDR-clients).
```

Comment:
Redundant with the previous line, might read better as "we can add it to our list of clients in Wiki". The latter part being a link.

Otherwise people will think that they can suggest and be added to the README, whereas in reality they'd be added to the wiki page.